src/local/butler/py_unittest.py

```diff
@@ -253,6 +257,15 @@ def execute(args):
       (appengine_config)  # pylint: disable=pointless-statement
     except ImportError:
       print('Note: unable to import appengine_config.')
+
+    # google.auth uses App Engine credentials based on importability of
```

Comment:
Can we call modules. _patch_appengine_modules_for_bots ?

Reply:
That will be removed very shortly once SDK is removed from Docker image/bot environments, so duplicating here.